serialization/rdf.md

```diff
@@ -2,10 +2,19 @@
 
 SPDX data can be serialized in RDF. This can be saved in a variety of formats, like XML, JSON-LD, Turtle, etc.
 
-1. The namespace for SPDX v3 is `http://spdx.org/rdf/v3`
+1. The namespace for SPDX v3 is `https://rdf.spdx.org/v3`
```

Comment:
In the current spec (version 2.X) the namespace also is a URL pointing to the documentation.

If we change it to `rdf.spdx.org`, we would need to register the subdomain through the LF.

Is this something we want to do? It make take a bit of work and time to get through the LF IT - but I can make an attempt at it if we want to change it.

Reply:
I opened a separate issue to track